### DISCUSSION TITLE
Is there way for Shell_gpt to run shell command directly without confirmation.

### DISCUSSION THREAD

**Discussion by sudhireer868** [TheR1D/shell_gpt#346] [General]
Is there way or setting in Shell_gpt to run shell command directly without waiting for user confirmation with below options.
[E]xecute, [D]escribe, [A]bort

Is there way to Execute or pass "e" to sgpt shell command.

**Comment by TheR1D** [comment]
It is possible in the [latest release](https://github.com/TheR1D/shell_gpt/releases/tag/1.1.0) with function calls.
